challenge_3/c/boilerup_brian/src/majority.c

```diff
@@ -0,0 +1,60 @@
+#include <stdio.h>
+
+int getMode(int *, int);
+int isMaj(int, int *, int);
+
+void printMaj(int set[], int size){
+    int mode = getMode(set, size);
+    int maj = isMaj(mode, set, size);
+    
+    if (maj == 0){
+        printf("Majority element not found\n");
+    }
+    else{
+        printf("Majority element is %d\n", mode);
+    }
+}
+
+int getMode(int set[], int size){
+    int i;
+    int mode_index = 0;
+    int count = 1;
+    for(i = 1; i < size; i++){
+        if(set[i] == set[mode_index]){
+            count ++;
+        }

```

Comment:
Its a minor detail, but from what I have seen many professional programmers do is simply do `} else {` instead of ```
}
else {
```

Which seems odd because you can save some lines by doing the first format. Is there a reason why you dont?